pkg/host/internal/kernel/kernel.go

```diff
@@ -522,6 +522,34 @@ func (k *kernel) InstallRDMA(packageManager string) error {
 	return nil
 }
 
+func (k *kernel) GetRDMASubsystem() (string, error) {
+	log.Log.Info("GetRDMASubsystem(): retrieving RDMA subsystem mode")
+	chrootDefinition := utils.GetChrootExtension()
+
+	stdout, stderr, err := k.utilsHelper.RunCommand("/bin/sh", "-c", fmt.Sprintf("%s /usr/bin/rdma system show", chrootDefinition))
+	if err != nil && len(stderr) != 0 {
+		log.Log.Error(err, "GetRDMASubsystem(): failed to get RDMA subsystem mode", "stdout", stdout, "stderr", stderr)
+		return "", err
+	}
+
+	// Example of an =output: netns shared copy-on-fork on
+	subsystem := strings.Fields(stdout)[1]
+	return subsystem, nil
```

Comment:
please add a len validation here, it's always hard to work with command outputs

Reply:
no need this anymore since we use netutils